src/main/java/nl/novi/techiteasymodel/controllers/TelevisionController.java

```diff
@@ -0,0 +1,41 @@
+package nl.novi.techiteasymodel.controllers;
+
+import nl.novi.techiteasymodel.models.Television;
+import nl.novi.techiteasymodel.repositories.TelevisionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+
+    @Autowired
+    private TelevisionRepository repos;
+
+    private ArrayList<Television> televisions;
+
+
+    @GetMapping("")
+    public ResponseEntity<String> getTelevisions(){
+        return ResponseEntity.ok("television" + repos.findAll());
+    }
+
+    @GetMapping("/{id}")
+    ResponseEntity<Object> getTelevisionWithId(@PathVariable Long id) {
+
+                return ResponseEntity.ok(repos.findById(id));
```

Comment:
Zie onderstaande uitwerking. Je geeft nu een `Optinal` terug.  En de `repos.findbyID(id) moet in de `ok().body( HIER )` staan.

```suggestion
                 Optional<Television> television = televisionRepository.findById(id);

        // Check of de optional empty is. Het tegenovergestelde alternatief is "television.isPresent()"
        if (television.isEmpty()){

            // Als er geen television in de optional staat, roepen we hier de RecordNotFoundException constructor aan met message.
            throw new RecordNotFoundException("No television found with id: " + id );

        } else {
            // Als er wel een television in de optional staat, dan halen we die uit de optional met de get-methode.
            Television television1 = television.get();

            // Return de television en een 200 status
            return ResponseEntity.ok().body(television1);
```